content/docs/04-clusters/03-edge/11-deploy-cluster.md

```diff
@@ -0,0 +1,856 @@
+```
+
+The cloning process can take up to 3-4 minutes to finish, and display the following output.
+<br />
+
+```bash coloredLines=6-6 hideClipboard
+# Sample output
+Cloned VM demo-1
+Powering on VM demo-1
+Powering on VirtualMachine:vm-9919... OK
+Getting UUID demo-1
+Edge Host ID   VM demo-1 : edge-97f2384233b498f6aa8dec90c3437c28
+```
+
+The output on the terminal will display the Edge host ID for all VMs. VMs will use this host ID to auto-register themselves with Palette. 
+<br />
+
+<InfoBox>
+
+Copy the Edge host ID for future reference and manual registration if the auto registration fails, for any reason, later in this tutorial. 
+
+</InfoBox>
+
+
+Note that the **edge-native/vmware/clone_vm_template/** directory in the container has the following files:
+- **deploy-edge-host.sh** - Provisions the VMs.
+- **delete-edge-host.sh** - Deletes the VMs.
+- **setenv.sh** - Defines the GOVC environment variables, the number of VMs, a prefix string for the VM name, and the VM template name. Most of the GOVC environment variables point to the default environment variables you have defined in the **.goenv** file. 
+
+Here is the **setenv.sh** file content for your reference; however, you do not have to modify these configuration in this tutorial. 
+<br />
+
+```bash hideClipboard
+#!/bin/bash
+# Number of VMs to provision
+export NO_OF_VMS=3
+export VM_PREFIX="demo"
+export INSTALLER_TEMPLATE="palette-edge-template"
+
+#### DO NOT MODIFY BELOW HERE ####################
+# GOVC Properties
+export GOVC_URL="https://${vcenter_server}"     # Use HTTPS. For example, https://vcenter.company.com
+export GOVC_USERNAME="${vcenter_username}"
+export GOVC_PASSWORD="${vcenter_password}"
+export GOVC_INSECURE=1 #1 if insecure
+export GOVC_DATACENTER="${vcenter_datacenter}"
+export GOVC_DATASTORE="${vcenter_datastore}"
+export GOVC_NETWORK="${vcenter_network}"
+export GOVC_RESOURCE_POOL="${vcenter_resource_pool}"
+export GOVC_FOLDER="${vcenter_folder}"
+```
+
+The **setenv.sh** file customization is out of the scope of this tutorial. To change the number of VMs to provision, you must clone the [tutorials](https://github.com/spectrocloud/tutorials.git) repository and edit the **edge-native/vmware/clone_vm_template/setenv.sh** file, rebuild the image using these [local build steps](https://github.com/spectrocloud/tutorials/blob/main/docs/docker.md#local-builds) and use that image in the previous `docker run` command. 
+<br />
+
+
+# Deploy a Cluster
+After building the Edge artifacts and provisioning VMs as Edge hosts, the next step is to verify the host registration, create a cluster profile, and deploy a cluster. 
+<br />
+
+## Verify Host Registration
+Ideally, VMs should automatically register with Palette if you correctly embed the registration token in the ISO image. Log back into Palette to verify, and navigate to the left **Main Menu** > **Clusters** > **Edge Hosts** tab. You should see the VMs registered with Palette automatically.
```

Comment:
```suggestion
Open a web browser and log in to [Palette](https://console.spectrocloud.com). Navigate to the left **Main Menu** and select **Clusters**. Click on the **Edge Hosts** tab and verify the three VMs created are registered with Palette.
```
1. Let's be optimistic and confident in our product. 
2. This is the first time the user is doing something in Palette, so we need to tell them to log in.
3. Don't take shortcuts. Help the user find the right place in the UI by guiding with sentences.